flask/patched.py

```diff
@@ -0,0 +1,4 @@
+from gevent import monkey
+monkey.patch_all()
+
+from app import app
```

Comment:
This is cool! What does it do?

Reply:
It intercepts a bunch of common python functions and makes them able to run asynchronously. Magic basically. I'm not 100% sure if it is intercepting everything we want, but I'll look into it more later.

Reply:
That's super cool. Nice Work. Merging